tests/common/test_constants.py

```diff
@@ -10,7 +10,7 @@ class TestConstants:
     @pytest.mark.parametrize("line", ["melon = 'banAna'", "melon : 'banAna'", "melon := 'banAna'"])
     def test_separator_common_p(self, config: Config, file_path: pytest.fixture, line: str) -> None:
         pattern = Util.get_keyword_pattern("melon")
-        line_data = LineData(config, line, 1, file_path, pattern)
+        line_data = LineData(config, line, 1, file_path, info='dummy', pattern=pattern)
```

Comment:
For the coding style.. Just change double quote to single quote.

```suggestion
        line_data = LineData(config, line, 1, file_path, info="dummy", pattern=pattern)
```